azure-mgmt-datalake-analytics/azure/mgmt/datalake/analytics/account/data_lake_analytics_account_management_client.py

```diff
@@ -35,16 +35,24 @@ class DataLakeAnalyticsAccountManagementClientConfiguration(AzureConfiguration):
      identify Microsoft Azure subscription. The subscription ID forms part of
      the URI for every service call.
     :type subscription_id: str
+    :param resource_group_name: The name of the Azure resource group.
+    :type resource_group_name: str
+    :param account_name: The name of the Data Lake Analytics account.
+    :type account_name: str
     :param str base_url: Service URL
     """
 
     def __init__(
-            self, credentials, subscription_id, base_url=None):
+            self, credentials, subscription_id, resource_group_name, account_name, base_url=None):
```

Comment:
@ro-joowan This means your Swagger is incorrect and defines an incorrect parameter-location

Reply:
see https://github.com/Azure/azure-openapi-validator/issues/84 for details

Reply:
@lmazuel Correct.  I fixed this is a PR that came right after this one: https://github.com/Azure/azure-rest-api-specs/pull/2417